mmcif_ma.dic

```diff
@@ -603,7 +608,8 @@ save__ma_template_details.template_label_asym_id
 
 save__ma_template_details.template_label_entity_id
     _item_description.description
-;     The entity ID corresponding to the template.
+;     The entity ID corresponding to the template. Points to the template
+      entity that needs to be defined in the same data block/ ModelCIF file.
```

Comment:
The `template_label_entity_id` also corresponds to the `label_entity_id` at the source of the template. This description needs to be fixed.

Reply:
Should be fixed by a new commit. Commit should go into the pull request automatically.